portal/templates/redesign/register.html

```diff
@@ -85,4 +85,10 @@
     </div>
 </div>
 
+<script type="text/javascript" src="{% static 'portal/js/passwordStrength_new.js' %}"></script>
+<script>
+    var PASSWORD_FIELD_ID = '{{ signup_form.password.auto_id }}';
+    var PASSWORD_TYPE = 'TEACHER';
```

Comment:
So far it's because the independent student registration forms doesn't work so by default the password type is TEACHER. This will be revisited in Pull Request 19 which deals with the independent student register form.